compiler/parser/src/lexer.rs

```diff
@@ -683,9 +681,26 @@ where
             } else {
                 StringKind::Normal
             };
+
+            // Reference for string literal prefixes:
+            // https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals
```

Comment:
@youknowone Feel free to fix this if this is not what you meant by https://github.com/RustPython/RustPython/pull/4343#discussion_r1056854560